README.md

```diff
@@ -88,7 +131,5 @@ curl http://cs.sensiolabs.org/download/php-cs-fixer-v2.phar > /tmp/php-cs-fixer.
 php /tmp/php-cs-fixer.phar  fix -v --diff --using-cache=yes src/
 ```
 
-TODO: 
-- Examples
+TODO:
 - ResolverInterface still needed?
```

Comment:
can we decide on this one? for the sake of customizing, i think we should keep it.

Reply:
Too long ago, since I used it. If it helps customizing, sure, let's keep it.

Reply:
i removed the todo